src/selectors/events.js

```diff
@@ -1,5 +1,38 @@
 // @flow
+import { parse as parseDate, differenceInCalendarDays } from "date-fns";
 import type { State } from "../reducers";
+import type { Event, EventDays } from "../data/event";
+
+const locale = "en-GB";
+
+const groupByStartTime = (events: Event[]): EventDays => {
+  const sections = events
+    .sort((a: Event, b: Event) => {
+      const diff =
+        parseDate(a.fields.startTime[locale]) -
+        parseDate(b.fields.startTime[locale]);
+
+      return diff / Math.abs(diff);
```

Comment:
Why do we need to divide here?

Reply:
The sorting callback expects `-1`, `0` or `1` back. This was a quick way to normalise if the difference is more than one day

Reply:
Ah, my bad, it doesn't care about the magnitude, probably thinking about a different language... Even better.